### PR TITLE
Port prefixdevname to fedora

### PR DESCRIPTION
**Pull request by rvykydal**

### CODE REVIEW

File: pyanaconda/modules/network/installation.py [hunk 1]
```diff
@@ -112,7 +114,8 @@ class NetworkInstallationTask(Task):
 """.strip()
 
     def __init__(self, sysroot, disable_ipv6, overwrite,
-                 network_ifaces, ifname_option_values):
+                 network_ifaces, ifname_option_values,
+                 configure_prefixdevname):
```
Comment (jkonecny12) [inline pyanaconda/modules/network/installation.py]: I don't really agree with the variable name. I don't think we should use the same name in dracut which is inconsistent with the rest of the code here.

File: pyanaconda/modules/network/installation.py [hunk 2]
```diff
@@ -254,6 +262,19 @@ def _copy_dhclient_config_files(self, root, network_ifaces):
             dhclient_file = self.DHCLIENT_FILE_TEMPLATE.format(device_name)
             self._copy_file_to_root(root, dhclient_file)
 
+    def _copy_prefixdevname_files(self, root):
+        """Copy prefixdevname persistent conifuration to target system.
```
Comment (jkonecny12) [inline pyanaconda/modules/network/installation.py]:
```suggestion
        """Copy prefixdevname persistent configuration to target system.
```

File: tests/nosetests/pyanaconda_tests/module_network_test.py [hunk 3]
```diff
@@ -1158,3 +1160,16 @@ def apply_boot_options_test(self):
...
+        mocked_kernel_args = {'ifname': "bla bla"}
+        self.assertEqual(
+            self.network_module._is_using_persistent_device_names(mocked_kernel_args),
+            False
+        )
+        mocked_kernel_args = {'net.ifnames.prefix': "net"}
+        self.assertEqual(
+            self.network_module._is_using_persistent_device_names(mocked_kernel_args),
+            True
+        )
```
Comment (jkonecny12) [inline tests/nosetests/pyanaconda_tests/module_network_test.py]: I don't think this test is what should be here. Point is to test the use case, that means that we are trying to achieve something by the call of something. This test is testing only our protected function which is only a part in the whole solution and should be hidden as uninteresting part of the whole test. 
In general testing of private API is not what should be tested.

File: pyanaconda/modules/network/installation.py (hunk 1 above)
Comment (rvykydal) [reply pyanaconda/modules/network/installation.py]: Renamed

File: pyanaconda/modules/network/installation.py (hunk 2 above)
Comment (rvykydal) [reply pyanaconda/modules/network/installation.py]: Fixed

File: tests/nosetests/pyanaconda_tests/module_network_test.py (hunk 3 above)
Comment (rvykydal) [reply tests/nosetests/pyanaconda_tests/module_network_test.py]: Removed. After merging the branch I'll open a PR for update of installation task unit tests that should cover also the private method.

**Pull request merged**